site/index.ts

```diff
@@ -159,10 +161,23 @@ domReady(() => {
             },
             imageUpload: imageUploadOptions,
             externalPlugins: [StackSnippetsPlugin],
-        });
+        };
+
+        const editorInstance = new StacksEditor(place, content.value, options);
+
         // set the instance on the window for developers to poke around in
         // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-explicit-any
         (window as any)["editorInstance"] = editorInstance;
+
+        const secondEditorInstance = new StacksEditor(
```

Comment:
I'm really cool with having a demo page where there are two editor instances since this is not the first bug around multiple editor instances on the same page. I recommend we add the "two editors on the same page" scenario on another site, though. See the `site/variants` directory? That's where we've got a bunch of other demo pages. Let's add the "two editors" scenario there (can just copy the `empty.html` view as a starting point, that should help reduce a lot of clutter). Also let's make sure to link to the new "two editors" page in the "Settings" menu in the top-right corner :)

Reply:
I've added a variant named dualeditors, added it to settings and modified index.ts to check if there is a second editor element on the page to avoid null exceptions.

Is it ok how I've done it or did I miss anything?